### PR TITLE
Setup wagtail

### PR DESCRIPTION
**Pull request by dalepotter**
Sets up a blank Django & Wagtail instance as a base for future development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+```
+# Set-up and activate virtual environment
+python3 -m venv pyenv
+source pyenv/bin/activate
+
+# Enter into project directory
+cd iati
+
+# Install requirements
+pip install -r requirements-dev.txt
```
Comment (allthatilk) [inline README.md]: We need to think about making this consistent throughout our code base. Suggest we stick to requirements_dev.txt.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+
+# Create a local PostgreSQL database (with appropriate user permissions)
+# settings/base.py expects username 'myusername' and password 'mypassword'
+# You can also use settings/local.py to set database credentials for your local instance
```
Comment (allthatilk) [inline README.md]: Suggest adding a link to the relevant Django documentation about overriding base.py settings.

File: iati/iati/settings/base.py [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        },
+    },
+]
+
+WSGI_APPLICATION = 'iati.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/2.0/ref/settings/#databases
+
```
Comment (allthatilk) [inline iati/iati/settings/base.py]: There's no need for this to be here if we add a local.py example file and leave base.py settings as is to be overridden with postgresql by an actual local.py. This also gives users an idea about how to use the local.py file.

File: iati/requirements-dev.txt [hunk 4]
```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
```
Comment (allthatilk) [inline iati/requirements-dev.txt]: File name change suggested above.

File: README.md (hunk 1 above)
Comment (dalepotter) [reply README.md]: pyIATI uses `requirements-dev.txt`, but most other repos have `requirements_dev.txt`.

Comment (dalepotter) [reply README.md]: 25dd37f

File: iati/requirements-dev.txt (hunk 4 above)
Comment (dalepotter) [reply iati/requirements-dev.txt]: 25dd37f

File: README.md (hunk 2 above)
Comment (dalepotter) [reply README.md]: The best I've found is https://github.com/wagtail/wagtaildemo/blob/master/wagtaildemo/settings/local.py.example#L1-L3 - do you think this is sufficient...?

File: iati/iati/settings/base.py (hunk 3 above)
Comment (dalepotter) [reply iati/iati/settings/base.py]: Makes sense, and is in line with the approach taken by wagtail in https://github.com/wagtail/wagtaildemo/blob/master/wagtaildemo/settings/local.py.example

Will move this to a `local.py.example` file and adjust the docs.

Comment (dalepotter) [reply iati/iati/settings/base.py]: 425260f

File: README.md (hunk 1 above)
Comment (allthatilk) [reply README.md]: We really need to fix consistency on this one. Suggest adding to monday projects to look at.

File: README.md (hunk 2 above)
Comment (allthatilk) [reply README.md]: I can't see anything about databases in base.py now. Maybe add something super basic along the same lines as wagtail https://github.com/wagtail/wagtaildemo/blob/master/wagtaildemo/settings/base.py#L18

Comment (dalepotter) [reply README.md]: Have reenabled SQLite as the default database in 091fe5e

**Pull request merged**